docs/src/main/sphinx/connector/redis.rst

```diff
@@ -50,20 +50,21 @@ Configuration properties
 
 The following configuration properties are available:
 
-=================================   ==============================================================
-Property Name                       Description
-=================================   ==============================================================
-``redis.table-names``               List of all tables provided by the catalog
-``redis.default-schema``            Default schema name for tables
-``redis.nodes``                     Location of the Redis server
-``redis.scan-count``                Redis parameter for scanning of the keys
-``redis.key-prefix-schema-table``   Redis keys have schema-name:table-name prefix
-``redis.key-delimiter``             Delimiter separating schema_name and table_name if redis.key-prefix-schema-table is used
-``redis.table-description-dir``     Directory containing table description files
-``redis.hide-internal-columns``     Controls whether internal columns are part of the table schema or not
-``redis.database-index``            Redis database index
-``redis.password``                  Redis server password
-=================================   ==============================================================
+======================================  ==============================================================
+Property Name                           Description
+======================================  ==============================================================
+``redis.table-names``                   List of all tables provided by the catalog
+``redis.default-schema``                Default schema name for tables
+``redis.nodes``                         Location of the Redis server
+``redis.scan-count``                    Redis parameter for scanning of the keys
+``redis.key-prefix-schema-table``       Redis keys have schema-name:table-name prefix
+``redis.key-delimiter``                 Delimiter separating schema_name and table_name if redis.key-prefix-schema-table is used
+``redis.table-description-dir``         Directory containing table description files
+``redis.table-description-cache-ttl``   The cache time for redis table description files
+``redis.hide-internal-columns``         Controls whether internal columns are part of the table schema or not
+``redis.database-index``                Redis database index
+``redis.password``                      Redis server password
+======================================  ==============================================================
```

Comment:
Please extract a commit to change the table width to improve diff.

Reply:
hi @ebyhr, I have divided two commits, do you think this is okay?

Reply:
@polaris6 The commit order is different. "Change the table width in redis doc" should be first without a new property.

Reply:
hi @ebyhr, I resubmitted it, do you think this is ok?